superdesk/activity.py

```diff
@@ -228,7 +228,9 @@ def add_activity(
     return activity
 
 
-def notify_and_add_activity(activity_name, msg, resource=None, item=None, user_list=None, **data):
+def notify_and_add_activity(
+    activity_name, msg, resource=None, item=None, user_list=None, users_with_enabled_email_notifications=None, **data
```

Comment:
would be good to rename `user_list` as well it's not obvious who are those users